Xamarin.Forms.Core/RelativeBindingSourceMode.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Xamarin.Forms
+{
+	public enum RelativeBindingSourceMode
+	{
+		// TODO? (much tricker than the other three)
+		// PreviousData = 0,
```

Comment:
Is this intentionally commented out?

Reply:
Yes I meant it as a placeholder in case someone wanted to tackle that down the road.